frontend/src/lib/components/LemonModal/LemonModal.tsx

```diff
@@ -28,6 +28,8 @@ export interface LemonModalProps {
     closable?: boolean
     /** Expands the modal to fill the entire screen */
     fullScreen?: boolean
+    /** set the z-index of the modal high enough that it is above (almost) all other content */
+    bringToFront?: boolean
```

Comment:
Hmm, I wonder why is this a special case? As a future user of `LemonModal`, I wouldn't know when/why should this be used, in a way that doesn't break UI

Reply:
Ah, yes, this is either genius or lazy 🤣 

modals are set slightly behind pop-ups so that a modal can have a select in it. But here we have a modal launched from a popup the pop-up can therefore overlap the modal.

I considered adding `modal-over-popup` or similar, but then what if I want a modal launched from a pop-up that has a select in it? So I thought I'd mimic part of how a lot of applications handle this and have `bring to front`

Maybe (for now?) it's enough to expand on that comment...